src/main/java/utils/DataProvider.java

```diff
@@ -22,6 +22,7 @@
     public DataProvider(SymBotClient botClient) {
         STANDARD_URI_SCHEMES.add("http");
         STANDARD_URI_SCHEMES.add("https");
+        STANDARD_URI_SCHEMES.add("quickws");
```

Comment:
Hum, that sounds very specific to be added like that to the SDK. It would be better to have a new configuration property called `"supportedUriSchemes"` which would have as default value `["http", "https"]`. Then the bot developer could decide to add new ones like : 
```json
{
    ...
    "supportedUriSchemes": ["http", "https", "quickws"]
}
```
![Screenshot 2020-05-18 at 18 34 23](https://user-images.githubusercontent.com/39826516/82237603-371aac00-9936-11ea-8ae3-f6e01c3a0225.png)